packages/lib/freeEmailDomainCheck/checkIfFreeEmailDomain.ts

```diff
@@ -0,0 +1,43 @@
+import { freeEmailDomainsThatStartWithANumberObject, freeEmailDomains } from "./freeEmailDomains";
+
+export const checkIfFreeEmailDomain = (email: string) => {
+  const emailDomain = email.split("@")?.[1];
```

Comment:
? is probably not needed
```suggestion
  const emailDomain = email.split("@").[1];
```